src/components/details/Details.tsx

```diff
@@ -19,16 +19,16 @@ export const Details = () => {
         audio: "",
       },
     ],
-    meanings=[
-
-    ],
-    sourceUrls=[],
+    meanings = [],
+    sourceUrls = [],
   } = data[0];
   const audio = useMemo(
     () => phonetics.filter((item: any) => item.audio !== ""),
     [data]
   );
-  const newAudio = new Audio(audio[0].audio);
+
+  let newAudio: HTMLAudioElement;
```

Comment:
you can use this line of code instead 

`  const newAudio : HTMLAudioElement = audio && new Audio(audio[0]?.audio);`